aws_secretsmanager_caching/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws_secretsmanager_caching"
-version = "1.2.1"
+version = "1.2.2"
```

Comment:
Do non-backwards compatible changes in the Cargo.toml (removal of a feature) warrant a major version bump? Genuine question.

Reply:
Well it would break consumers of the library that use the feature in their `Cargo.toml`

```toml
aws_secretsmanager_caching = { version = "1", features = ["prefer-post-quantum"] }
```